images/base/scripts/common/setup.sh

```diff
@@ -90,9 +109,14 @@ apt-get install -y build-essential libtool
 #./configure
 ./configure --prefix=/usr
 
-make
-make check
-make install
+if [ x$MACHINE = xs390x ]
+then
+    echo FIXME: protobufs wont compile on 390, missing atomic call
```

Comment:
I am fine with leaving it as a FIXME if the only implication is the lack of protoc compilation.  I might have guessed there would be run-time implications as well w.r.t. the package installing .so files that the golang protobuf integration depended on if I didn't know better.  However, the evidence is to the contrary since we know it builds and can run the tests, etc.